### DISCUSSION TITLE
How to create column headers and input boxes in Table widget?

### DISCUSSION THREAD

**Discussion by ShaunKulesa** [fltk-rs/fltk-rs#980] [Q&A] [answered]
I have a table and how do I add input in each box and column headers?
```
let mut table = Table::new(0, 30, 300, 500, "database name");
table.set_rows(30);
table.set_cols(5);
```

**Comment by MoAlyousef** [comment, marked as answer]
You can try the fltk-table crate:
https://crates.io/crates/fltk-table

You can set cell and header values using set_cell_value and set_<row/col>_header_value(), respectively. You can check the examples in the repo:
https://github.com/fltk-rs/fltk-table

There's also a tutorial here:
https://www.youtube.com/watch?v=pVJ8Yq1kDGs
